app/src/main/java/org/schabi/newpipe/player/Player.java

```diff
@@ -3754,6 +3754,9 @@ public boolean onKeyDown(final int keyCode) {
             case KeyEvent.KEYCODE_SPACE:
                 if (isFullscreen) {
                     playPause();
+                    if (isPlaying()) {
```

Comment:
Why is this needed?

Reply:
So that the behaviour is consistent with manually pausing (keeps the controls visible).

Reply:
Okay but why is this only the case for the full-screen player?

Reply:
The `if (isFullscreen)` was already there so I just left it. But I've tested and it works fine even when not in fullscreen so I've removed the `if (isFullscreen)` in https://github.com/TeamNewPipe/NewPipe/pull/7341/commits/29348411520f56f07865c6deef722ae107b9a205